app/controllers/users_controller.rb

```diff
@@ -39,16 +43,16 @@ def update
     end
   end
 
-  private
+   private
 
-  def user_params
-    params.require(:user).permit(:username, :password, :current_password, :email, :cell)
-  end
+   def user_params
+     params.require(:user).permit(:username, :password, :current_password, :email, :cell)
+   end
 
-  def check_for_correct_user
-    if (!current_user || current_user.id != params[:id].to_i)
-      flash[:warning] = "Oops, you visited the wrong page."
-      redirect_to root_path
-    end
-  end
+   def check_for_correct_user
+     if !current_user || current_user.id != params[:id].to_i
+       flash[:warning] = "Oops, you visited the wrong page."
+       redirect_to root_path
+     end
```

Comment:
We can get rid of this method. It's not necessary, there are better ways of ensuring a user is only able to see their own information as opposed to someone else's.